### PR TITLE
dev-db/unixODBC: add 2.3.11, move to EAPI 8

### PR DESCRIPTION
**Pull request by scardracs**
Signed-off-by: Marco Scardovi <scardracs-gentoo@proton.me>

### CODE REVIEW

File: dev-db/unixODBC/unixODBC-2.3.11.ebuild [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~loong ~m68k ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~sparc-solaris ~sparc64-solaris ~x64-solaris ~x86-solaris"
+IUSE="fastvalidate +minimal odbcmanual readline static-libs stats threads unicode"
+
+RDEPEND="
+	|| (
+		dev-libs/libltdl:0[${MULTILIB_USEDEP}]
+		>=sys-devel/libtool-2.4.2-r1[${MULTILIB_USEDEP}]
+	)
+	>=sys-libs/readline-6.2_p5-r1:0=[${MULTILIB_USEDEP}]
```
Comment (thesamesam) [inline dev-db/unixODBC/unixODBC-2.3.11.ebuild:22]: You've added e.g. USE=readline but not made the deps conditional?

Comment (scardracs) [reply dev-db/unixODBC/unixODBC-2.3.11.ebuild:22]: It was already present in older version so I haven't touched these deps at all

Comment (scardracs) [reply dev-db/unixODBC/unixODBC-2.3.11.ebuild:22]: Uh, I've forgot it. I'll fix it this evening

File: dev-db/unixODBC/unixODBC-2.3.11.ebuild [hunk 2]
```diff
@@ -0,0 +1,69 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit multilib-minimal
+
+DESCRIPTION="complete ODBC driver manager"
+HOMEPAGE="http://www.unixodbc.org/"
```
Comment (scardracs) [inline dev-db/unixODBC/unixODBC-2.3.11.ebuild:9]: I hate that but actually it is http

**Pull request closed without merging**